### DISCUSSION TITLE
Pokedex

### DISCUSSION THREAD

**Discussion by viridianab** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#471] [Show and tell]
Hola :-)
Me basé en la versión de Kalos pero agregando unas animaciones y sonidos para fuera más interactiva.

https://viridianab.github.io/pokedex/

**Comment by JavierMendez-Coder** [comment]
¡Te quedó **increible**! 👏👏
Si aceptas retroalimentación, poder utilizar la tecla enter para realizar la busqueda estaría muy bien.

**Reply by viridianab** [reply]
¡Muchas gracias!
Intentaré agregar lo del enter :)

**Comment by Edgarcastrop** [comment]
Te quedo muy bien, disculpa como haces en el caso de los movimientos para que se pueda hacer scroll en el cuadro?

**Reply by viridianab** [reply]
Gracias :)
Y para esa parte encerré el `<ul>` de los movimientos en un div, este tiene una altura definida y además le agregué un overflow: scroll para que se pueda ver todo el resto del div

**Comment by sergiobps** [comment]
Woow, quedo bien padre! solo le falta lo del enter como comentaron antes y estaría perfecta. Muy detallada y colorida.

**Reply by viridianab** [reply]
Muchas gracias :-)

**Comment by lk4eiji** [comment]
Te quedó muy bien!. Me gustó mucho el detalle de los li para los tipos. Me gustaría implementarlo en mi pokedex, ¿me darías el permiso para hacerlo?.

**Reply by viridianab** [reply]
Gracias y sí, no hay problema :D

**Comment by SujeiRodriguez** [comment]
todos con sus pokedex bien chidas :'v

**Comment by khisus19** [comment]
Está genial! Te felicito.

**Comment by edu18RR** [comment]
Te quedo impresionante.
